### ISSUE TITLE
URL check fails without error message

### ISSUE THREAD

**Issue by dosch** [wallabag/wallabagger#70]
Using the Wallabagger FIrefox Plugin 1.3

Adding my Wallabag URL fails. 
With https and http `mydomain.com/read/web`
Box around url gets red border after clicking 'Check URL'

![screen shot 2017-04-05 at 21 14 30](https://cloud.githubusercontent.com/assets/5372610/24722724/1b8049d2-1a45-11e7-95c1-6ac6145c7dd4.png)

But nothing else happens...

**Comment by Simounet** [comment]
Is it the URL you use to access wallabag? Do you have any htaccess/credentials to be used? Did you removed the http protocol from the URL you added?

**Comment by dosch** [comment]
Yes it is the same URL.
I have no htaccess/credentials to be used.
I did remove the http(s) from the url.

**Comment by dosch** [comment]
Found it. 

Removing the trailing `/` solved the issue.

Thank you @Simounet for replying so fast.

**Comment by Simounet** [comment]
Hmmmm, I thought we handled that so I reopen this to fix it.

**Comment by nmaggioni** [comment]
The same thing happens to me with a DDNS domain, custom port and self-signed SSL certificate: `https://[DOMAIN].ddns.net:32370` fails.

**Comment by Simounet** [comment]
Did you removed the trailing slash?

**Comment by nmaggioni** [comment]
Yes, the URL is just as typed above. Without SSL it seems to works fine, but I need it for the de-duplication feature.

**Comment by Simounet** [comment]
On your browser, do you need to confirm the certificate before seeing the wallabag page?

**Comment by nmaggioni** [comment]
I do, it is a self-signed cert served via NGINX. Is it unsupported by the Chrome extension? The Android app handles it fine when checking the option to ignore certificate validation in the settings.

**Comment by Simounet** [comment]
It is not handled here right now. Can you open a new issue please?

**Comment by akwala** [comment]
I see this issue when setting the options for the Wallabagger 1.3 addon in Firefox.
![wallabagger1 3-fx-options-urlcheck](https://cloud.githubusercontent.com/assets/1786/25096980/9028099e-2370-11e7-94da-a4786faf2f8a.png)

**Comment by Simounet** [comment]
Hi there,
Sorry for the late reply. Is it fixed or do you still have this issue?

**Comment by akwala** [comment]
@Simounet yes, I still have this issue.

**Comment by Simounet** [comment]
Do you have any blocking ressource addon loaded (Privacy Badger like)? Can you try it on a fresh profile?

**Comment by akwala** [comment]
[RequestPolicyContinued](https://github.com/RequestPolicyContinued/requestpolicy) -- added rule to allow requests to app.wallabag.it. Currently it's a temporary rule, I'm curious to see how often it'll be needed.

**Comment by Simounet** [comment]
So can I consider your problem solved?

**Comment by akwala** [comment]
@Simounet yes, thank you.

**Comment by pasbec** [comment]
I have the same problem with Firefox 52.6.0 with Version 1.4.5. I've removed any trailing slashes. I use self-hosted version of wallabag without any problems in android, via web-interface and other firefox browsers.

Is there any chance to force accept the url? Why is there no error message?

Any help is appreciated. Thanks!

**Comment by Simounet** [comment]
Hello,
Do you have any extension that can interfere with it? Can you try it on a fresh profile?
Can you follow these step:
- enable the debug mode into the wallabagger settings
- go to about:debugging#addons
- check in Enable add-on debugging
- click on wallabag debug link
- click on ok to accept incoming connections
- add some page through the context-menu
- give us the entries on the console tab of the new window opened

**Comment by pasbec** [comment]
Hello,

I finally fixed it!

I have a lot of extensions installed, so I couldn't really tell if interference was an issue with my profile. However a fresh profile AND another firefox with the same addons manually installed did work. So it seemed to be some broken addon-settings. Another indication for this assumption was that wallabager always showed some (missing) token as expired in its settings - I don't know why, since everything was originlally correctly configured and nothing has changed with my wallabag token or the firefox, except for regular firefox updates. Somehow wallabager suddenly stopped working presumably due to broken settings.

Uninstalling wallabager in the broken profile did however not remove the addon-settings. Therefore I finally deleted the folder `browser-extension-data\{7a7b1d36-d7a4-481b-92c6-9f5427cb9eb1}` in my profile folder of firefox and restarted the browser. Afterwards I could re-enter the data for my wallabag-server and the problem was fixed.

Anyway, thank you very much for your advice with the fresh profile and sorry for not using the debugging mode!
